iothub/service/src/Exceptions/IotHubServiceErrorCode.cs

```diff
@@ -159,6 +159,13 @@ public enum IotHubServiceErrorCode
         /// </summary>
         ConfigurationNotFound = 404301,
 
+        // Request Timeout - 408
+
+        /// <summary>
+        /// The client did not send a request within the time the server was expecting the request.
+        /// </summary>
+        GenericRequestTimeout = 408000,
```

Comment:
I'd prefer we just name this `RequestTimeout`. I don't find that the word "Generic" means much to me as a user.